circular_saw.lua

```diff
@@ -366,15 +365,17 @@ function circular_saw.on_construct(pos)
 	local meta = minetest.get_meta(pos)
 	local fancy_inv = default.gui_bg..default.gui_bg_img..default.gui_slots
 	meta:set_string(
+		--For some reason the translation does not work with @n in this part
```

Comment:
Have you tried
``` S("Input\nmaterial")
```
and
```
Input@nmaterial=Material de@nentrada
```
 ? This usually worked on other projects I updated translation system on…

Reply:
That is what I have tried, but it does not work, it is strange because in infotext if it works

Reply:
Oh, it seems related to this bug https://github.com/minetest/minetest/issues/7450. And this is specific to labels… Maybe you should had a FIXME with a reference to this issue in your comment. Your workaround is fine.